packages/aws_logging_cloudwatch/sdk.yaml

```diff
@@ -0,0 +1,6 @@
+apis:
+  logs:
+  - com.amazonaws.cloudwatchlogs#PutLogEvents
+  - com.amazonaws.cloudwatchlogs#DescribeLogGroups
+  - com.amazonaws.cloudwatchlogs#DescribeLogStreams
```

Comment:
Are these needed in the main pkg or just for tests?

Reply:
they maybe needed for the main pkg, depends if we need to call DescribeLogGroups/DescribeLogStreams to check if they exists. I think it is best to remove them now and we add when we are sure we need them.